components/NameTagForm.tsx

```diff
@@ -80,9 +106,19 @@ export function NameTagForm({
               )}
             />
             </div>
-          <input type="submit" className="submit-btn" />
```

Comment:
Let's switch the position of "display nametag" switch and "save name tag" button, since it is more intuitive to save right after creating/updating nametag.